### ISSUE TITLE
Use LocalDrive `mirror()` method

### ISSUE THREAD

**Issue by josephmturner** [RangerMauve/hyper-sdk-folder-sync-example#4]
In v1.4.0 as of two days ago, LocalDrive exposes a `mirror()` method, which calls mirror-drive in basically the same way this package does, except it doesn't filter anything out.

For simplicity's sake, I can use that instead. Wdyt?

**Comment by RangerMauve** [comment]
Yeah that might be useful. 👍 I see it's in both hyperdrive and localdrive so it should be safe. A PR would be welcome or I can do it if you'd like. 😁

**Comment by josephmturner** [comment]
Done.
